src/tests/suite/dfs_test.c

```diff
@@ -166,6 +167,16 @@ main(int argc, char **argv)
 		tests[new_idx] = '\0';
 	}
 
+	/** if writing XML, force all ranks other than rank 0 to use stdout to avoid conflicts */
+	cmocka_message_output = getenv("CMOCKA_MESSAGE_OUTPUT");
+	if (rank != 0 && cmocka_message_output && strcasecmp(cmocka_message_output, "xml") == 0) {
+		rc = setenv("CMOCKA_MESSAGE_OUTPUT", "stdout", 1);
```

Comment:
Unclean because master has `d_setenv` but 2.4 does not

Reply:
@knard-intel is there a plan to port that change on 2.4? if so, we probably need to fix this later?

Reply:
From what I know, it is not planned to backport the PRs related to the ticket [DAOS-14532](https://daosio.atlassian.net/browse/DAOS-14532)

Reply:
+1. I don't think we need to backport DAOS-14532